rust/hyperlane-base/src/types/multisig.rs

```diff
@@ -10,6 +11,18 @@ use hyperlane_core::{
 };
 
 use crate::{CheckpointSyncer, CoreMetrics};
+/// Weights are scaled by 1e10 as 100%
```

Comment:
does this matter from the agent's POV? feels like a smart contract impl detail that has no impact on agent operations

Reply:
It's moreso context for someone looking at the logs with a weight = 12300000 and not understanding what the scale is. Most likely, looks like an absolute stake amount from this perspective, so it can be confusing.